scripts/sync_milestones.js

```diff
@@ -12,6 +12,7 @@ var ghlib = require('./ghlib.js')
 var P = require('bluebird')
 
 var ACTIVE_FEATURE_LABELS = {
+  "designin": true,
```

Comment:
My bad, I pushed a quick update to this while @vladikoff was reviewing and I think this slipped in during the race; I fixed it in https://github.com/mozilla/fxa/commit/a5f1bebad1c1f95da050ef3554b67b9eb560b14a